sentry_streams/tests/adapters/arroyo/test_steps.py

```diff
@@ -32,13 +53,37 @@ def make_msg(payload: Any, route: Route, offset: int) -> Message[Any]:
         return Message(
             BrokerValue(
                 payload=RoutedValue(route=route, payload=payload),
-                partition=Partition(Topic("test_topic"), 0),
+                partition=TEST_PARTITION,
                 offset=offset,
                 timestamp=datetime(2025, 1, 1, 12, 0),
             )
         )
 
 
+def make_value_msg(payload: Any, route: Route, offset: int) -> Message[Any]:
```

Comment:
crazy enough, I added this to main right before you yesterday 

seems identical to the current version in main, but just wanted to point that out in case theres a conflict